frontend/src/Frontend/UI/Dialogs/Send.hs

```diff
@@ -335,48 +335,62 @@ sendConfig model initData = Workflow $ do
           displayImmediateFeedback (updated decoded) cannotBeReceiverMsg
             $ either (const False) (\ka -> _txBuilder_accountName ka == fromName && _txBuilder_chainId ka == fromChain)
 
-          (_, amount, _) <- mkLabeledInput True "Amount" uiGasPriceInputField
-            (def & inputElementConfig_initialValue .~ (maybe "" tshow mInitAmount))
-            -- We're only interested in the decimal of the gas price
-            <&> over (_2 . mapped . mapped) (\(GasPrice (ParsedDecimal d)) -> d)
+          let
+            balance = _account_status fromAcc ^? _AccountStatus_Exists . accountDetails_balance . to unAccountBalance
+
+            gasInputWithMaxButton cfg = mdo
+              let projectAmount = over (_2 . mapped . mapped) (\(GasPrice (ParsedDecimal d)) -> d)
+                  attrs = ffor useEntireBalance $ \u -> "disabled" =: ("disabled" <$ u)
```

Comment:
There is no real style change on the disabled. Probably good to add some CSS to make it look proper disabled, because it looks confusing to me in brave. Or, display the actual balance in the field when max is toggled.

Reply:
Looks like we're keeping this one with the same style as the read-only fields